server/test/tokensIntegrationTests.js

```diff
@@ -0,0 +1,114 @@
+const chai = require('chai');
+const assert = chai.assert;
+const chaiHttp = require('chai-http');
+const sinon = require('sinon');
+const app = require('../app');
+const db = require('../lib/db');
+
+chai.use(chaiHttp);
+chai.should();
+
+describe("Token Integration Tests: ", () => {
+    describe("Create an Account - POST & GET", () => {
+        var organization = "New Test Organization";
+        var token = "";
+
+        before(() => {
+
+        });
+
+        beforeEach(() => {
+
+        });
+
+        it("POST token/", (done) => {
```

Comment:
I think the POST and GET should be one test case (in an it() )

Reply:
why is that?